index.html

```diff
@@ -104,9 +108,216 @@ <h2 class="color-heading" id="heading-2">
           </div>
         </div>
       </div>
     </section>
+
+    <section class="work-section">
+      <div class="opening-heading">
+        <h2 class="work-heading">My Recent Work</h2>
+      </div>
+      <div class="container-box container-grid">
+        <div class="section-container1 section-box item-grid">
+          <ul class="first-work display-flex-1">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project1.png"
+                alt="Tonic Project: Personalized reading selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">
+                Tonic Project: Personalized Reading
+              </h3>
+            </li>
+            <li class="display-flex-width languages-used">
+              <ul class="languages language-flex">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+            </li>
+            <li class="display-flex-width btn-project-1">
+              <button class="action-button" type="button">See Project</button>
+            </li>
+          </ul>
+        </div>
+
+        <div class="section-container2 section-box item-grid">
+          <ul class="first-work display-flex-2">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project2.png"
+                alt="Multi Post Stories: Personalized Stories selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">Multi Post Stories</h3>
+            </li>
+            <li class="display-flex-width languages-used">
+              <ul class="languages language-flex">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+            </li>
+            <li class="display-flex-width btn-project-1">
+              <button class="action-button" type="button">See Project</button>
+            </li>
+          </ul>
+        </div>
+
+        <div class="section-container3 section-box item-grid">
+          <ul class="first-work display-flex-3">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project3.png"
+                alt="Health and Fitness Project: Personalized Stories selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">Health and Fitness Project</h3>
```

Comment:
- [ ] Use `a` a tag that will redirect to the project specifications like live demo link or on GitHub.